src/react-native/13-making-http-requests/making-http-requests.md

```diff
@@ -106,17 +114,151 @@ If you’ve implemented the solution correctly, the tests will pass when you run
+
+✏️ Create **src/services/pmo/restaurant/interfaces.ts** and update it to be:
+
+@diff ../../../exercises/react-native/13-http-requests/01-solution/src/services/pmo/restaurant/interfaces.ts ../../../exercises/react-native/13-http-requests/02-problem/src/services/pmo/restaurant/interfaces.ts only
+
+✏️ Update **src/screens/CityList/CityList.tsx** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/01-solution/src/screens/CityList/CityList.tsx ../../../exercises/react-native/13-http-requests/02-problem/src/screens/CityList/CityList.tsx only
+
+### Verify 2
+
+✏️ Update **src/services/pmo/restaurant/hooks.test.ts** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/01-solution/src/services/pmo/restaurant/hooks.test.ts ../../../exercises/react-native/13-http-requests/02-problem/src/services/pmo/restaurant/hooks.test.ts only
+
+✏️ Update **src/screens/CityList/CityList.test.tsx** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/01-solution/src/screens/CityList/CityList.test.tsx ../../../exercises/react-native/13-http-requests/02-problem/src/screens/CityList/CityList.test.tsx only
+
+### Exercise 2
+
+TODO
+
+### Solution 2
+
+If you’ve implemented the solution correctly, the tests will pass when you run `npm run test`!
+
+<details>
+<summary>Click to see the solution</summary>
+
+✏️ Update **src/services/pmo/restaurant/hooks.ts** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/02-problem/src/services/pmo/restaurant/hooks.ts ../../../exercises/react-native/13-http-requests/02-solution/src/services/pmo/restaurant/hooks.ts only
+
 ✏️ Update **src/screens/CityList/CityList.tsx** to be:
 
-@diff ../../../exercises/react-native/13-http-requests/01-problem/src/screens/CityList/CityList.tsx ../../../exercises/react-native/13-http-requests/01-solution/src/screens/CityList/CityList.tsx only
+@diff ../../../exercises/react-native/13-http-requests/01-problem/src/screens/CityList/CityList.tsx ../../../exercises/react-native/13-http-requests/02-solution/src/screens/CityList/CityList.tsx only
+
+</details>
+
+## Objective 3: Create API request helper
+
+### Setup 3
+
+✏️ Create **src/services/pmo/api/api.ts** and update it to be:
+
+@sourceref ../../../exercises/react-native/13-http-requests/03-problem/src/services/pmo/api/api.ts
+
+✏️ Create **src/services/pmo/api/index.ts** and update it to be:
+
+@sourceref ../../../exercises/react-native/13-http-requests/03-problem/src/services/pmo/api/index.ts
+
+### Verify 3
+
+✏️ Update **src/services/pmo/restaurant/hooks.test.ts** to be:
+
+@diff ../../../exercises/react-native/13-http-requests/02-solution/src/services/pmo/restaurant/hooks.test.ts ../../../exercises/react-native/13-http-requests/03-problem/src/services/pmo/restaurant/hooks.test.ts only
+
+### Exercise 3
+
+TODO
```

Comment:
same here 😅

Reply:
Yep, creating exercises is in the content portion